src/routes/safe/components/Apps/hooks/useAppList.ts

```diff
@@ -1,30 +1,42 @@
 import { useState, useEffect, useCallback } from 'react'
 import { loadFromStorage, saveToStorage } from 'src/utils/storage'
-import { APPS_STORAGE_KEY, getAppInfoFromUrl, getAppsList, getEmptySafeApp } from '../utils'
-import { AppData } from '../api/fetchSafeAppsList'
+import { APPS_STORAGE_KEY, getAppInfoFromUrl, getEmptySafeApp } from '../utils'
+import { AppData, fetchSafeAppsList } from 'src/logic/configService'
 import { SafeApp, StoredSafeApp, SAFE_APP_FETCH_STATUS } from '../types'
 import { getNetworkId } from 'src/config'
+import enqueueSnackbar from 'src/logic/notifications/store/actions/enqueueSnackbar'
+import { NOTIFICATIONS } from 'src/logic/notifications'
+import { useDispatch } from 'react-redux'
 
 type UseAppListReturnType = {
   appList: SafeApp[]
   removeApp: (appUrl: string) => void
-  staticAppsList: AppData[]
+  apiAppsList: AppData[]
+  isLoading: boolean
 }
 
 const useAppList = (): UseAppListReturnType => {
   const [appList, setAppList] = useState<SafeApp[]>([])
-  const [staticAppsList, setStaticAppsList] = useState<AppData[]>([])
+  const [apiAppsList, setApiAppsList] = useState<AppData[]>([])
+  const dispatch = useDispatch()
+  const [isLoading, setIsLoading] = useState(false)
 
   useEffect(() => {
     const loadAppsList = async () => {
-      const remoteAppsList = await getAppsList()
-      setStaticAppsList(remoteAppsList)
+      setIsLoading(true)
+      try {
+        const result = await fetchSafeAppsList()
+        setApiAppsList(result && result?.length ? result : apiAppsList)
+      } catch (err) {
+        dispatch(enqueueSnackbar(NOTIFICATIONS.SAFE_APPS_FETCH_ERROR_MSG))
```

Comment:
Do we need to log an error here/send it to sentry?

Reply:
As a general rule, I wouldn't log API errors. They should be logged on the API side.

Reply:
But what if an API error leads to another error or bug on the client? I think something like a non-tracked error won't hurt

Reply:
Sure, logging to the console is OK.

Reply:
Should i log the error to the console then?

Reply:
Yes, using the CodedException logError function.